charts/metallb/values.yaml

```diff
@@ -262,6 +262,7 @@ speaker:
   logLevel: info
   tolerateMaster: true
   memberlist:
+    # When enabled: false, speaker pods must run on all nodes, no nodeSelector
```

Comment:
```suggestion
    # When enabled: false, the speaker pods must run on all nodes
```

Reply:
we can enforce the mutual exclusion with helm rules in the templates

Reply:
we have already a policy, is that what you mean by rule?
 https://github.com/metallb/metallb/blob/651eb25245a054ab76698ecc399a040b97267851/charts/metallb/policy/speaker.rego#L11-L16 which does not even allow to disable the memberlist. Not sure the rules/policies are working though

Reply:
nope, it's in the chart. Try to look for `fail` in the charts, there are a few examples.